src/useValidation/validationState.ts

```diff
@@ -0,0 +1,5 @@
+export type ValidationState = {
```

Comment:
Something I've been doing recently is including a reactive of all the refs a composition returns. This is pretty close to that but without a few of the refs. Its a nice devx to be able to work with a reactive vs refs when able and I think it makes sense for this composition to expose everything both as a ref and in this reactive state. 

prefect-design can just use the pieces it needs (I'd have prefect design's `state` still just be the limited set that it actually uses).

Reply:
can you give me an example of where you do this? I'm curious how to define it's type and what to name the property

Reply:
That's what I would do